src/main/flight/pid_autotune.c

```diff
@@ -122,16 +123,31 @@ void autotuneUpdateState(void)
         DISABLE_FLIGHT_MODE(AUTO_TUNE);
     }
 }
+
+static void blackboxLogAutotuneEvent(adjustmentFunction_e adjustmentFunction, int32_t newValue)
+{
+#ifndef BLACKBOX
+    UNUSED(adjustmentFunction);
+    UNUSED(newValue);
+#else
+    if (feature(FEATURE_BLACKBOX)) {
```

Comment:
Maybe the head should add "#include "config/feature.h"" to avoid the warning info.